acquisitions/src/main/resources/thunderjet/mod-orders/features/independent-acquisitions-unit-for-ordering-and-receiving.feature

```diff
@@ -0,0 +1,720 @@
+    * def acqUnitId1 = callonce uuid10
+    * def acqUnitId2 = callonce uuid11
+    * def acqUnitId3 = callonce uuid12
+    * def acqUnitMembershipId1 = callonce uuid13
+    * def acqUnitMembershipId2 = callonce uuid14
+    * def acqUnitMembershipId3 = callonce uuid15
+    * def acqUnitMembershipId4 = callonce uuid16
+    * def acqUnitMembershipId5 = callonce uuid17
+    * def locationId = callonce uuid18
+
+  ## Acq units inheritance from Order checks
+  Scenario: Create Order with acqUnit1, Create PO Line, check that acqUnit1 was inherited from Order to Title
+    # 1. Create acq unit
+    * configure headers = headersAdmin
+    Given path 'acquisitions-units/units'
+    And headers headersAdmin
+    And request
+    """
+    {
+      "id": '#(acqUnitId1)',
+      "protectUpdate": true,
+      "protectCreate": true,
+      "protectDelete": true,
+      "protectRead": true,
+      "name": "testAcqUnit"
+    }
+    """
+    When method POST
+    Then status 201
+
+    # 2. Create acq unit membership
+    * configure headers = headersAdmin
+    Given path 'acquisitions-units/memberships'
+    And headers headersAdmin
+    And request
+    """
+      {
+        "id": '#(acqUnitMembershipId1)',
+        "userId": "00000000-1111-5555-9999-999999999992",
+        "acquisitionsUnitId": "#(acqUnitId1)"
+      }
+    """
+    When method POST
+    Then status 201
+
+    # 3. Create a fund and budget
+    * configure headers = headersAdmin
+    * call createFund { 'id': '#(fundId)', 'ledgerId': '#(globalLedgerId)'}
+    * callonce createBudget { 'id': '#(budgetId)', 'fundId': '#(fundId)', 'allocated': 1000, 'statusExpenseClasses': [{'expenseClassId': '#(globalPrnExpenseClassId)','status': 'Active'}]}
+    * configure headers = headersUser
+
+    # 4. Create a composite order
+    * configure headers = headersUser
+    Given path 'orders/composite-orders'
+    And request
+    """
+    {
+      id: '#(orderId1)',
+      vendor: '#(globalVendorId)',
+      orderType: 'One-Time',
+      "acqUnitIds": ['#(acqUnitId1)']
+    }
+    """
+    When method POST
+    Then status 201
+
+    # 5. Create an order line
+    Given path 'orders/order-lines'
+    * def poLine = read('classpath:samples/mod-orders/orderLines/minimal-order-line.json')
+    * set poLine.id = poLineId1
+    * set poLine.purchaseOrderId = orderId1
+    * set poLine.fundDistribution[0].fundId = fundId
+    And request poLine
+    When method POST
+    Then status 201
+
+    # 6. Open the order
+    Given path 'orders/composite-orders', orderId1
+    When method GET
+    Then status 200
+
+    * def order = $
+    * set order.workflowStatus = 'Open'
+
+    Given path 'orders/composite-orders', orderId1
+    And request order
+    When method PUT
+    Then status 204
+
+    # 6. Check that acqUnit1 was inherited from Order to Title
+    * configure headers = headersUser
+    Given path 'orders/titles'
+    And param query = 'poLineId==' + poLineId1
+    When method GET
+    Then status 200
```

Comment:
Need to add check here that title has this field: acqUnitIds: ["acqUnitId1"]

Reply:
Fixed